plugin/src/cli_utils/mod.rs

```diff
@@ -0,0 +1,139 @@
+use kubectl_plugin::resources;
+use plugin::ExecuteOperation;
+use clap::Parser;
+use std::ops::Deref;
+pub(crate) mod localpvlvm;
+pub(crate) mod localpvzfs;
+pub(crate) mod localpvhostpath;
+pub(crate) mod clusterinfo;
+
+#[derive(Parser, Debug)]
+#[group(skip)]
+pub(crate) struct CliArgs {
+    #[clap(flatten)]
+    args: resources::CliArgs
+}
+
+impl Deref for CliArgs {
+    type Target = plugin::CliArgs;
+
+    fn deref(&self) -> &Self::Target {
+        &self.args
+    }
+}
+
+/// Storage engines supported.
+#[derive(Parser, Debug)]
+pub(crate) enum Operations {
+    /// Mayastor specific commands
```

Comment:
Missing `.` and `C`apitalization

```suggestion
    /// Mayastor specific commands.
```

Reply:
Done